go.mod

```diff
@@ -4,7 +4,7 @@ go 1.18
 
 require (
 	github.com/CosmWasm/wasmd v0.24.0
-	github.com/cosmos/cosmos-sdk v0.45.3
+	github.com/cosmos/cosmos-sdk v0.45.4
```

Comment:
What is the reason for this? Is this necessary?

We usually upgrade our SDK fork in the replace directive here:
https://github.com/osmosis-labs/osmosis/blob/d8918936a898bcc982147f8e42d096a0f9a473db/go.mod#L271

Reply:
I'll change this back - I think I got this as a part of pulling from main before making these changes but I could be wrong